packages/rum-recorder/test/utils.ts

```diff
@@ -51,3 +51,26 @@ export class MockWorker implements DeflateWorker {
     this.pendingMessages.length = 0
   }
 }
+
+export function waitCalls<F extends jasmine.Func>(
+  spy: jasmine.Spy<F>,
```

Comment:
```suggestion
  requestSpy: jasmine.Spy<F>,
```
it could make the implementation clearer

Reply:
It is not for "requests" anymore though. Any functions that are called asynchronously would be a good candidate for this.

Reply:
fair enough